sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -1188,4 +1183,42 @@ class JoinSuite extends QueryTest with SharedSparkSession with AdaptiveSparkPlan
         classOf[BroadcastNestedLoopJoinExec]))
     }
   }
+
+  test("SPARK-32494: Null Aware Anti Join Optimize Support Multi-Column") {
+    withSQLConf(SQLConf.OPTIMIZE_NULL_AWARE_ANTI_JOIN.key -> "true",
+      SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key -> Long.MaxValue.toString) {
+      // positive not in subquery case
+      var joinExec = assertJoin((
+        "select * from testData where (key, key + 1) not in (select * from testData2)",
```

Comment:
Could you please use uppercases for SQL keywords where possible? e.g., `SELECT * FROM testData...`

Reply:
updated